.appveyor.yml

```diff
@@ -4,33 +4,48 @@
 
 version: '{branch}.{build}'
 
-install:
-- cmd: choco feature enable -n=allowGlobalConfirmation
-- cmd: mkdir %APPVEYOR_BUILD_FOLDER%\bin
+# TODO: Add options for testing. See https://github.com/nikic/php-ast/blob/master/.appveyor.yml for an example
+environment:
+  matrix:
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_EXT_VERSION: '7.1'
+      VC_VERSION: 14
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      PHP_EXT_VERSION: '7.2'
+      VC_VERSION: 15
+
+init:
+    - SET PATH=c:\projects\php;C:\projects\composer;%PATH%
 
 branches:
   only:
     - master
 
+install:
+- cmd: choco feature enable -n=allowGlobalConfirmation
+- cmd: mkdir %APPVEYOR_BUILD_FOLDER%\bin
+
 build_script:
 - cmd: >-
-    "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\vcvars32.bat"
+    SET BINARY_TOOLS_BASENAME=php-sdk-binary-tools-20110915.zip
 
-    appveyor DownloadFile http://windows.php.net/downloads/php-sdk/php-sdk-binary-tools-20110915.zip
+    curl -fsS http://windows.php.net/downloads/php-sdk/%BINARY_TOOLS_BASENAME% -o %BINARY_TOOLS_BASENAME%
 
-    7z x -y php-sdk-binary-tools-20110915.zip -oC:\projects\php-sdk
+    7z x -y %BINARY_TOOLS_BASENAME% -oC:\projects\php-sdk
 
     C:\projects\php-sdk\bin\phpsdk_setvars.bat
 
-    git clone --depth=1 --branch=PHP-7.1 https://github.com/php/php-src C:\projects\php-src
+    git clone --depth=1 --branch=PHP-%PHP_EXT_VERSION% https://github.com/php/php-src C:\projects\php-src
 
     mkdir C:\projects\php-src\ext\runkit
 
     xcopy %APPVEYOR_BUILD_FOLDER% C:\projects\php-src\ext\runkit /s /e /y
 
-    appveyor DownloadFile http://windows.php.net/downloads/php-sdk/deps-7.1-vc14-x86.7z
+    SET DEPS_BASENAME=deps-7.1-vc%VC_VERSION%-x86.7z
```

Comment:
This is failing in php 7.2+. See https://wiki.php.net/internals/windows/stepbystepbuild_sdk_2
and https://github.com/Microsoft/php-sdk-binary-tools